tests/cypress/support/commands.js

```diff
@@ -11,6 +11,7 @@ require('cypress-localstorage-commands');
 require('../plugins/compareImages/compareImagesCommand');
 
 let selectedValueGlobal = '';
+const projectSubsetFieldValue = 'Test';
```

Comment:
Guess it can be better to make this variable as a default `createAnnotationTask` parameter. It can be helpful in future tests.

Reply:
Reworked.